README.md

```diff
@@ -107,6 +107,16 @@ The official build automatically builds and tests all the assemblies in this rep
 
 Make sure to investigate any ApiCompat failures that arise, as they only show up when packing.
 
+Make sure to also update the version of any projects that depend on the one you're updating, and do this recursively as needed. Only a handful of projects are dependencies of others:
+
+| Project                                | Dependents |
+|----------------------------------------|------------|
+| System.Buffers                         | <ul><li>System.Buffers</li><li>System.Memory</li><li>System.Net.WebSockets.WebSocketProtocol</li></ul> |
+| System.Memory                          | <ul><li>Microsoft.IO.Redist</li><li>System.Net.WebSockets.WebSocketProtocol</li></ul> |
+| System.Numerics.Vectors                | <ul><li>System.Memory</li><li>System.Net.WebSockets.WebSocketProtocol</li></ul> |
+| System.Runtime.CompilerServices.Unsafe | <ul><li>Microsoft.IO.Redist</li><li>System.Memory</li><li>System.Net.WebSockets.WebSocketProtocol</li><li>System.Threading.Tasks.Extensions</li></ul> |
+| System.Threading.Tasks.Extensions      | <ul><li>System.Net.WebSockets.WebSocketProtocol</li></ul> |
```

Comment:
This duplicates the source dependencies.  Instead of listing all this information out, can we just enforce this with a build target like https://github.com/dotnet/runtime/blob/0fdf85d5c50997999f77e019bc26272212856f53/eng/packaging.targets#L359-L361

Reply:
@ericstj @ViktorHofer I can work on that. Do you mind if I submit such build target in a separate PR? That way we don't have to block the version bumps on it.

And I'll revert the readme changes here.